.github/workflows/format.yml

```diff
@@ -0,0 +1,23 @@
+name: Format
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Format
+        run: |
+          npm ci
+          npm run format
```

Comment:
Since `nodecloud` uses yarn instead of the npm client, this needs to be replaced with,

```
yarn
yarn format
```

Reply:
Also, the format command needs to be updated in the `package.json` as mentioned in the article. I will look more into that and let you know.

Reply:
Ok.

Reply:
you need to add these line in the `package.json` file

"format": "prettier --write ."

@lumenCodes

Reply:
As a dev dependency?

Reply:
Done @rajitha1998